app/questionnaire/questionnaire_schema.py

```diff
@@ -898,6 +907,60 @@ def _get_rules_section_dependencies(
 
         return rules_section_dependencies
 
+    def _populate_calculated_summary_section_dependencies(self) -> None:
+        for section in self.get_sections():
+            for block in self.get_blocks_for_section(section):
+                sources = self._get_dictionaries_with_key(
+                    "source", block, ignore_keys=["when"]
+                )
+
+                calculated_summary_sources = [
+                    source
+                    for source in sources
+                    if source.get("source") == "calculated_summary"
```

Comment:
Minor, use bracket access instead of `.get()` when the key is mandatory

Reply:
Updated 👍